PVE_SCHOLAR_DT.lua

```diff
@@ -101,6 +101,14 @@ function Profile:SkillTable(Data,Target,ClassTypeID)
 	local ShouldApplyBio = self.TargetBuff2(Target,{179,189,1895},3,"Missing",PlayerID)
 	
     local AetherflowCD = self.GetActionCD(166,true)
+	
+	local AOEType = { ["Filter"] = "Enemy", ["Name"] = "Circle", ["TargetPoint"] = PlayerPOS, ["AOERange"] = 35, ["MaxDistance"] = 0, ["LineWidth"] = 0, ["Angle"] = 0, }
+	local EnemiesAroundSelf = self.EntityInCount(AOEType)
+	local InTrashMobs = EnemiesAroundSelf >= 2
```

Comment:
I initially had this as PartyAggroCount >= 2, but PartyAggroCount was always 0 in duty support, unsure if that's a minion bug, but this works too.

Reply:
PartyAggroCount is working fine here. Where is it not working for you as it would be optimal to keep it using this rather than an additional calculation